hub-templates/basehub/values.yaml

```diff
@@ -67,8 +72,6 @@ jupyterhub:
     hook:
       enabled: false
   proxy:
-    https:
-      enabled: false
```

Comment:
I guess this is now `false` by default?

Reply:
Yepp!

Reply:
Yeah but it was also repeated down in the same file

Reply:
Gotcha!